sections/multirow.liquid

```diff
@@ -0,0 +1,438 @@
+        {
+          "value": "large",
+          "label": "t:sections.multirow.settings.image_height.options__4.label"
+        }
+      ],
+      "default": "small",
+      "label": "t:sections.multirow.settings.image_height.label"
+    },
+    {
+      "type": "select",
+      "id": "desktop_image_width",
+      "options": [
+        {
+          "value": "small",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__1.label"
+        },
+        {
+          "value": "medium",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__2.label"
+        },
+        {
+          "value": "large",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__3.label"
+        }
+      ],
+      "default": "medium",
+      "label": "t:sections.multirow.settings.desktop_image_width.label",
+      "info": "t:sections.multirow.settings.desktop_image_width.info"
+    },
+    {
+      "type": "select",
+      "id": "heading_size",
+      "options": [
+        {
+          "value": "h2",
+          "label": "t:sections.multirow.settings.heading_size.options__1.label"
+        },
+        {
+          "value": "h1",
+          "label": "t:sections.multirow.settings.heading_size.options__2.label"
+        },
+        {
+          "value": "h0",
+          "label": "t:sections.multirow.settings.heading_size.options__3.label"
+        }
+      ],
+      "default": "h1",
+      "label": "t:sections.multirow.settings.heading_size.label"
+    },
+    {
+      "type": "select",
+      "id": "text_style",
+      "options": [
+        {
+          "value": "body",
+          "label": "t:sections.multirow.settings.text_style.options__1.label"
+        },
+        {
+          "value": "subtitle",
+          "label": "t:sections.multirow.settings.text_style.options__2.label"
+        }
+      ],
+      "default": "body",
+      "label": "t:sections.multirow.settings.text_style.label"
+    },
+    {
+      "type": "select",
+      "id": "button_style",
+      "options": [
+        {
+          "value": "primary",
+          "label": "t:sections.multirow.settings.button_style.options__1.label"
+        },
+        {
+          "value": "secondary",
+          "label": "t:sections.multirow.settings.button_style.options__2.label"
+        }
+      ],
+      "default": "secondary",
+      "label": "t:sections.multirow.settings.button_style.label"
+    },
+    {
+      "type": "select",
+      "id": "desktop_content_position",
+      "options": [
+        {
+          "value": "top",
+          "label": "t:sections.multirow.settings.desktop_content_position.options__1.label"
+        },
+        {
+          "value": "middle",
+          "label": "t:sections.multirow.settings.desktop_content_position.options__2.label"
+        },
+        {
+          "value": "bottom",
+          "label": "t:sections.multirow.settings.desktop_content_position.options__3.label"
+        }
+      ],
+      "default": "top",
```

Comment:
```suggestion
      "default": "middle",
```
This would suit better different length of content and balance better the white space between the rows. It becomes even more obvious with this type of predefined and opinionated layout this section offers.